src/app/campaign-details/campaign-details.component.spec.ts

```diff
@@ -68,7 +68,7 @@ describe('CampaignDetailsComponent', () => {
       countries: ['United Kingdom'],
       currencyCode: 'GBP',
       donationCount: 4,
-      endDate: new Date(),
```

Comment:
These were not realistic tests - data fetched by HTTP using standard JSON deserialize will never contain Date objects. Adjusted types force us to write something more realistic, with strings here instead.